### ISSUE TITLE
Spacarlight only prints 5 digits for the node locations

### ISSUE THREAD

**Issue by yuitert** [precision-utwente/spalight#4]
This is not sufficiently accurate to get gradient based optimization stable.

In my spacar version line 280 for user defined nodes and 358 for intermediate nodes.

I would propose to go to maximum accuracy which comes at a cost of readibility

**Comment by Marijnn** [comment]
Makes sense to increase precision of the numbers printed. We will have to investigate what a sensible precision is, i.e. check what the dat-file parser of Spacar is set to support.
